src/__tests__/Media.spec.js

```diff
@@ -1,130 +1,122 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { render, screen } from '@testing-library/react';
 import { Media } from '..';
+import {
+  testForChildrenInComponent,
+  testForDefaultTag,
+} from '../testUtils';
 
 describe('Media', () => {
-  it('should render a div tag by default', () => {
-    const wrapper = shallow(<Media />);
 
-    expect(wrapper.type()).toBe('div');
+  it('should render a div tag by default', () => {
+    testForDefaultTag(Media, 'div');
   });
 
   it('should render an h4 tag by default for heading', () => {
-    const wrapper = shallow(<Media heading />);
-
-    expect(wrapper.type()).toBe('h4');
+    render(<Media heading data-testid="media"/>);
+    expect(screen.getByTestId('media').tagName.toLowerCase()).toBe('h4');
   });
 
-  it('should render an a tag by default Media with an href', () => {
-    const wrapper = shallow(<Media href="#" />);
 
-    expect(wrapper.type()).toBe('a');
+  it('should render an a tag by default Media with an href', () => {
+    render(<Media href="#" data-testid="media"/>);
+    expect(screen.getByTestId('media').tagName.toLowerCase()).toBe('a');
   });
 
   it('should render an img tag by default for object', () => {
-    const wrapper = shallow(<Media object />);
-
-    expect(wrapper.type()).toBe('img');
+    render(<Media object data-testid="media"/>);
+    expect(screen.getByTestId('media').tagName.toLowerCase()).toBe('img');
   });
 
   it('should render an img tag by default Media with a src', () => {
-    const wrapper = shallow(<Media src="#" />);
-
-    expect(wrapper.type()).toBe('img');
+    render(<Media src="#" data-testid="media"/>);
+    expect(screen.getByTestId('media').tagName.toLowerCase()).toBe('img');
   });
 
   it('should render a ul tag by default for list', () => {
-    const wrapper = shallow(<Media list />);
-
-    expect(wrapper.type()).toBe('ul');
+    render(<Media list data-testid="media"/>);
+    expect(screen.getByTestId('media').tagName.toLowerCase()).toBe('ul');
   });
 
   it('should pass additional classNames', () => {
-    const wrapper = shallow(<Media className="extra" />);
-
-    expect(wrapper.hasClass('extra')).toBe(true);
+    render(<Media className="extra" data-testid="media" />);
+    expect(screen.getByTestId('media')).toHaveClass('extra');
   });
 
-  it('should render custom tag', () => {
-    const wrapper = shallow(<Media tag="main" />);
 
-    expect(wrapper.type()).toBe('main');
+  it('should render custom tag', () => {
+    render(<Media tag="main" data-testid="media"/>);
```

Comment:
hey @minimatrix thank you for the PR.

You can use the function `testForCustomTag` from testUtils for this test.

Reply:
@illiteratewriter - thanks for reviewing, I have amended that test to use the function from testUtils as recommended